contracts/contracts/rem.system/src/producer_pay.cpp

```diff
@@ -125,7 +137,7 @@ namespace eosiosystem {
             _producers.modify(prod, same_payer, [&](auto& p) {
                p.last_expected_produced_blocks_update = timestamp;
             });
-            _gstate.last_schedule[i] = std::make_pair(prod_name, prod.total_votes / _gstate.total_active_producer_vote_weight);
+            _gstate.last_schedule[i] = std::make_pair(prod_name, (std::floor(prod.total_votes * 10.0) / 10.0) / _gstate.total_active_producer_vote_weight);
```

Comment:
should we call update_pervote_shares() here?

Reply:
we can, but this will add one more loop over _gstate.last_schedule